core/rovecomm.py

```diff
@@ -73,6 +86,39 @@ def print(self):
         print('----------')
 
 
+class RoveCommCollection:
```

Comment:
Call this just RoveComm, and this should initialize a RoveCommUDP and RoveCommTCP instance. Also define a write function that takes a boolean "Reliable" parameter that determines if it should send over UDP and TCP.